src/rtextures.c

```diff
@@ -2073,6 +2073,65 @@ void ImageColorReplace(Image *image, Color color, Color replace)
 
     ImageFormat(image, format);
 }
+// Flood-fills from the given position
+void ImageFloodFill(Image *dst, int startPosX, int startPosY, Color color){
+    if ((dst->data == NULL) || (startPosX < 0) || (startPosX >= dst->width) || (startPosY < 0) || (startPosY >= dst->height)) return;
+    
+    int bpp = GetPixelDataSize(1,1,dst->format);
+    void *start_pixel_address = (dst->data + bpp*(startPosX + dst->width*startPosY));
+    
+    Color startColor = GetPixelColor(start_pixel_address, dst->format);
+    if (startColor.r == color.r && startColor.g == color.g && startColor.b == color.b && startColor.a == color.a) return;
+    
+    struct flood_fill_pixel_pos {
```

Comment:
Please, could you avoid additional structures definition?

Reply:
I assumed it would be local only. I think I was wrong? 
Changed it.